config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] Great job working on this milestone. Kindly remove this commented line of code and any other in your codebase. It is not professional to keep commented pieces of code in our codebase.